File: python/template/template_test.py
import template
import unittest

class TemplateTest(unittest.TestCase):

    def setUp(self):
        self.input_data = ""
        self.expected_output = ""

    def _check_function(self, func):
        self.assertEqual(func(self.input_data), self.expected_output)

    def test_1(self):
        self.input_data = "hola"
        self.expected_output = "hola"
        self._check_function(template.run)

File: java/template/TemplateTest.java
import static org.junit.Assert.assertEquals;
import org.junit.Test;
import org.junit.Before;

public class TemplateTest { 

    private int input_data;
    private int expected_output;
    private Template template;

    @Before
    public void setUp() {
        this.template = new Template();
    }

    @Test
    public void test1()  {
        input_data = 1;
        expected_output = 2;
        assertEquals(template.run(this.input_data), this.expected_output);
    }
}
